E-Commerce/readme.md

```diff
@@ -46,12 +47,147 @@ erDiagram
+ Mobile Phone X  |        249
+ Javascript Book |         41
+ Javascript Book |         40
+ Javascript Book |         39
+(5 rows)
+
+
+```
+
 - [ ] List all the products sold by suppliers based in the United Kingdom. The result should only contain the columns product_name and supplier_name
+
+```sql
+
+select product_name ,supplier_name                                                                                 from product_availability as pa INNER JOIN products as p on (pa.prod_id = p.id)                                                   INNER JOIN suppliers as s on( s.id = pa.supp_id)                                                                                  where s.country='United Kingdom';
+      product_name       | supplier_name 
+-------------------------+---------------
+ Javascript Book         | Argos
+ Super warm socks        | Argos
+ Coffee Cup              | Argos
+ Tee Shirt Olympic Games | Argos
+ Mobile Phone X          | Sainsburys
+ Le Petit Prince         | Sainsburys
+ Super warm socks        | Sainsburys
+ Coffee Cup              | Sainsburys
+ Ball                    | Sainsburys
+(9 rows)
+
+
+
+```
+
 - [ ] List all orders, including order items, from customer named Hope Crosby
+```sql
+
+select order_id,product_id, supplier_id, quantity                                                                  from customers as c INNER JOIN orders as o on (c.id =o.customer_id)                                                               INNER JOIN order_items as oi on (o.id = oi.order_id)                                                                              where name like 'Hope Crosby';
+ order_id | product_id | supplier_id | quantity 
+----------+------------+-------------+----------
+        4 |          1 |           1 |        1
+(1 row)
+
+
+
+
+
+```
+
 - [ ] List all the products in the order ORD006. The result should only contain the columns product_name, unit_price, and quantity
+```sql
+ SELECT p.product_name, pa.unit_price, oi.quantity 
+cyf_ecommerc-# FROM orders as o 
+cyf_ecommerc-# INNER JOIN order_items oi ON o.id = oi.order_id 
+cyf_ecommerc-# INNER JOIN products as p ON oi.product_id = p.id 
+cyf_ecommerc-# INNER JOIN product_availability as pa ON p.id = pa.prod_id  
+cyf_ecommerc-# WHERE o.order_reference = 'ORD006';
+   product_name   | unit_price | quantity 
+------------------+------------+----------
+ Coffee Cup       |          3 |        3
+ Coffee Cup       |          4 |        3
+ Coffee Cup       |          4 |        3
+ Coffee Cup       |          5 |        3
+ Javascript Book  |         40 |        1
+ Javascript Book  |         41 |        1
+ Javascript Book  |         39 |        1
+ Le Petit Prince  |         10 |        1
+ Le Petit Prince  |         10 |        1
+ Super warm socks |         10 |        3
+ Super warm socks |          5 |        3
+ Super warm socks |          8 |        3
+ Super warm socks |         10 |        3
+(13 rows)
```

Comment:
CORRECT - Again it would nice to see this without additional unnecessary text, but the SQL is right.